### PR TITLE
Fixing ADF orbital parser

### LINKED ISSUE #182: ADF orbital parser
**Describe the bug**
The current adf orbital parser fails when there are more than one irrep in the orbital energies table. The current test uses the `resource('adf-lu.out')` file which only has one irrep so it passes with no issues. When we use the file `resource('adf-nico4.out')` it fails as the dataframe is now picking up the irrep in the vectors column and we can't do the subtraction by 1.

**To Reproduce**
``` python
from exatomic.base import resource
from exatomic.adf import Output
ed = Output(resource('adf-nico4.out'))
ed.parse_orbital()
```

**Expected behavior**
No failures

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows/WSL
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by herbertludowieg**
Closes #182.
Added functionality to be able to parse the orbital energies when there is more than one irrep in the orbital energy table.

Currently the vectors are take as is from the output. Need to run tests to see if this messes up the orbital plotting.

### CODE REVIEW

File: exatomic/adf/output.py [hunk 1]
```diff
@@ -202,9 +202,36 @@ def parse_orbital(self):
...
             _re_orb_01: ['vector', 'occupation', 'energy', 'eV', 'dE']}
         key = _re_orb_00 if found[_re_orb_00] else _re_orb_01
-        start = stop = found[key][-1] + 5
-        while self[stop].strip(): stop += 1
-        df = self.pandas_dataframe(start, stop, cols[key])
+        start = stop = found[key][-1] + 4
+        starts = []
+        stops = []
+        irreps = []
+        while True:
```
Comment (herbertludowieg) [inline exatomic/adf/output.py]: I know this is a very dangerous thing to do, but I couldn't find another way to test that there is no error raised when intentionally converting a non-int string to a string.

Comment (tjduigna) [reply exatomic/adf/output.py:209]: It looks well bounded by `len(self)` so it's probably fine. Is line 214 necessary?

Comment (herbertludowieg) [reply exatomic/adf/output.py:209]: It is so we can catch any `ValueError` when trying to convert a string to an int. This comes from the way ADF arranges the symmetry as blocks with the symmetry element in a new line. Small example:
```
A
       A-symmetry orbital stuff
T
       T symmetry orbital stuff
```

**Comment by herbertludowieg** [comment]
I'll go ahead and take care of the print statements. I was just making sure that it wouldn't cause any issues with it, but you guys already disabled parsing the `momatrix` table if there is symmetry. Maybe that will be for a later PR.

**Comment by tjduigna** [comment]
Yeah, symmetrized momatrix is *always* out of scope lol. I think if you pull in master then appveyor will be happy?

**Pull request merged**